### DISCUSSION TITLE
Snapshot Quality

### DISCUSSION THREAD

**Discussion by ali86x** [arendst/Tasmota#21485] [Q&A]
Hello.
I  use tasmota Firmware on esp32 with Webcam.
Works nice so far.
But.....
WcResolution is set to 8 (1024x768) .
Snapshots and stream are only 640x480 ??
How can i change that?
Snapshot via IP/snapshot.jpg and stream via IP:81/stream

**Comment by magiva** [comment]
any updates here ?

**Reply by sfromis** [reply]
For XGA resolution instead of VGA, try `WcResolution 10` instead of 8.
